command/command.go

```diff
@@ -119,7 +120,11 @@ func (c *command) Run(args []string) int {
 	}
 
 	if err := c.runFunc(args); err != nil {
-		if err == errShowHelp {
+		if strings.Contains(err.Error(), errShowHelp.Error()) {
```

Comment:
The general approach here is really nice (using an error value to indicate we need to show help)!. I'm not a fan of the string comparison though, this is fragile. Take a look at the "cause" functionality of the errors pkg: https://github.com/pkg/errors#retrieving-the-cause-of-an-error.